.changeset/lemon-words-pump.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
+---
+ForestTypeExpensiveDebug now validates content against schema
+
+When opting into using [ForestTypeExpensiveDebug](https://fluidframework.com/docs/api/fluid-framework/#foresttypeexpensivedebug-variable) using [configuredSharedTree](https://fluidframework.com/docs/api/fluid-framework/#configuredsharedtree-function), the tree is now checked against the schema on load and after every edit.
```

Comment:
Nit: might be worth adding a quick example of configuring the SharedTree with this flag.

Reply:
Done